src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -154,6 +119,12 @@ function getDialogToolbarSelector(dialogName) {
     return ".wym_section.wym_tools.wym_buttons li a[name=" + dialogName + "]";
 }
 
+function getDialogKeyCombo(dialogName) {
+    return {
+        CreateLink: "ctrl+k"
+    }[dialogName] || null;
+}
```

Comment:
This returns `null` if the `dialogName` is not in this literal object expression.